elixir/apps/web/lib/web/live/actors/show.ex

```diff
@@ -142,18 +142,6 @@ defmodule Web.Actors.Show do
                 <div class="pb-4">
                   No authentication identities to display
                 </div>
-                <.add_button
-                  :if={is_nil(@actor.deleted_at) and @actor.type == :service_account}
-                  navigate={~p"/#{@account}/actors/service_accounts/#{@actor}/new_identity"}
-                >
-                  Create Token
-                </.add_button>
-                <.add_button
-                  :if={is_nil(@actor.deleted_at) and @actor.type != :service_account}
-                  navigate={~p"/#{@account}/actors/users/#{@actor}/new_identity"}
-                >
-                  Create Identity
-                </.add_button>
```

Comment:
I'm not opposed to removing this, I just happened to noticed in the commit message you referenced removing this to be more consistent.  Just wondering what consistency you had in mind?

Reply:
In these views, we have an `Add ...` button in the top-right, but then we were also showing a big action button in the table itself which is a bit redundant I think. Just a minor thing to improve visual consistency.

<img width="1217" alt="Screenshot 2023-12-21 at 7 57 46 AM" src="https://github.com/firezone/firezone/assets/167144/b0b386db-8d04-41bd-b4b8-913a09ccf049">
<img width="1209" alt="Screenshot 2023-12-21 at 7 56 46 AM" src="https://github.com/firezone/firezone/assets/167144/eb921df6-fa15-4431-aa87-dac57103209c">

Reply:
Or we could make it a link:
<img width="1208" alt="Screenshot 2023-12-21 at 7 59 18 AM" src="https://github.com/firezone/firezone/assets/167144/fc713294-ca1a-43e9-acd2-65f36907ece1">

Just thinking buttons should either be aligned-left or aligned-right. Body-level actions / navigation should probably be links for consistency.

Reply:
Gotcha.  Yeah, I'm totally fine with not having the buttons (or links for that matter) in the empty table view when we have the buttons to "Add" things up in the right hand corner.  It does seem a little redundant.

The only reason I asked this was because there are a lot of other views/tables throughout the app that do have the buttons in the empty table view, so in terms of consistency we might want to either remove all of them or none of them.

<img width="730" alt="Screenshot 2023-12-21 at 11 21 02 AM" src="https://github.com/firezone/firezone/assets/2646332/15b32192-9307-4447-841b-4d377ff33cc2">
<img width="655" alt="Screenshot 2023-12-21 at 11 21 21 AM" src="https://github.com/firezone/firezone/assets/2646332/5ff1c82c-4e60-4ac0-a1b3-756389c17bd6">
<img width="653" alt="Screenshot 2023-12-21 at 11 21 28 AM" src="https://github.com/firezone/firezone/assets/2646332/dc327de7-733a-477f-80aa-9ad204bbed58">

Reply:
Good catch! I was looking for all those. Let me revert that commit in this PR and open a new PR to fix those other ones.

Reply:
I think for UX we should keep the empty state buttons, even if duplicated they help users to know how to proceed right away, which is especially good when there is more than one button to the right of the title.